packages/components/tag/src/variants/filled.ts

```diff
@@ -46,4 +46,9 @@ export const filledVariants = [
     design: 'filled',
     class: tw(['bg-neutral', 'text-on-neutral']),
   },
+  {
+    intent: 'surface',
+    design: 'filled',
+    class: tw(['bg-surface', 'text-on-surface']),
```

Comment:
Do we still need this `tw()` util?

Reply:
Unfortunately, yes. Otherwise, we lose the autocompletion / intellisense. This is due to a limitation of the Tailwind ESLint plugin's typings, which cannot handle very large objects, if I recall correctly